IntegrationTests/lib.js

```diff
@@ -45,6 +45,8 @@ const startWasiTask = async (wasmPath) => {
     swift.setInstance(instance);
     // Start the WebAssembly WASI instance!
     wasi.start(instance);
+    instance.exports._initialize();
+    instance.exports.main();
```

Comment:
Why isn’t this called by e.g. `wasi.start`? Is there a way to make sure existing users will make this change when upgrading?

Reply:
That's because @wasmer/wasi doesn't support reactor. But even though WASI implementation supports it, users need to call `WASI.initialize` instead: https://nodejs.org/api/wasi.html#wasiinitializeinstance.

> These kinds are mutually exclusive;

https://github.com/WebAssembly/WASI/blob/59cbe140561db52fc505555e859de884e0ee7f00/legacy/application-abi.md#current-unstable-abi

And they (command and reactor) are exclusive, so there is no way to upgrade without changing entrypoints